spec.html

```diff
@@ -39905,8 +39905,8 @@
         <h1>Changes to Abstract Equality Comparison</h1>
         <p>The following steps are inserted after step 3 of the <emu-xref href="#sec-abstract-equality-comparison">Abstract Equality Comparison</emu-xref> algorithm:</p>
         <emu-alg>
-          1. If _x_ has an [[IsHTMLDDA]] internal slot and _y_ is either *null* or *undefined*, return *true*.
-          1. If _x_ is either *null* or *undefined* and _y_ has an [[IsHTMLDDA]] internal slot, return *true*.
+          1. If Type(_x_) is Object and _x_ has an [[IsHTMLDDA]] internal slot and _y_ is either *null* or *undefined*, return *true*.
```

Comment:
My feeling is "X has [[SomeSlot]] internal slot" implicitly carries an "if Type(_x_) is Object" but explicitness is fine here too (unless I'm missing the issue this is addressing).

Reply:
I don't think it does, unfortunately, given how many places in the spec precede internal slot checks with type checks.

Reply:
What @domenic said: The spec always tests the type before accessing internal slots.